docs/Ecosystem.md

```diff
@@ -8,6 +8,7 @@ title: "Ecosystem"
 - [Inputs and Fields](#inputs-and-fields)
 - [Translations](#translations)
 - [Data Providers](#data-providers)
+- [User Interface](#ui)
```

Comment:
missing entry for authorization management, that you've just asked. Also, it's confusing because the items and the sections don't follow the same order.